brion/plugin/compartmentReportBinary.cpp

```diff
@@ -706,23 +716,23 @@ bool CompartmentReportBinary::_parseMapping()
                 const uint64_t frameIndex =
                     j + ((info.dataOffset - _header.dataBlockOffset) /
```

Comment:
As long as there is a paranoid check that does not affect the performance, I have no preference here.
If you think logic_error is better, it's fine with me.